### ISSUE TITLE
no-unused-variable and static class members

### ISSUE THREAD

**Issue by scriby** [palantir/tslint#115]
```
class Test {
  static x = '123';
}
```

The no-unused-variable rule is triggering an error that x is not used. We have some enum-like classes that are formulated like the above, and all the entries may not be referenced by the program.

Should the no-unused-variable rule not apply to static class members? (Or at least an option)?
